src/constants/schema.ts

```diff
@@ -164,31 +164,114 @@ export type TTenantInvitationSchema = z.infer<typeof tenantInvitationSchema>;
 
 /**
  * The schema for the shareHouse creation form
+ * It is used to new sharehouse
  */
 export const shareHouseCreationSchema = shareHouseNameSchema.extend({
```

Comment:
Is it possible to simplify the shareHouseCreationSchema and shareHouseConfirmSchema a bit more?

Reply:
I agree with you. I'd like to do that in later refactor phases.